pages/common/git-branch.md

```diff
@@ -11,9 +11,9 @@
 
 `git branch -a`
 
-- Show the name of the current branch:
+- Find what branch a Git commit came from:
```

Comment:
```suggestion
- Find which branch a Git commit came from:
```

Reply:
Another suggetion:
```suggestion
- Show the branch a Git commit came from:
```

Reply:
```suggestion
- List what branches includes a specific Git commit in their history:
```

Reply:
A commit may be part of the history of multiple branches. What about this?

```suggestion
- List which branches include a specific Git commit in their history:
```